src/plugin.rs

```diff
@@ -11,6 +9,7 @@ use bevy::{
     utils::HashMap,
 };
 use rive_rs::Instantiate;
+use std::sync::Arc;
```

Comment:
This was probably moved by mistake.

Reply:
It wasn't, there was a line break I removed between the imports. Fmt alphabetically sorted the import after.